lib/vpc/models/DhcpConfig.rb

```diff
@@ -68,6 +68,7 @@ def populate!(aws)
       def diff(aws)
         diffs = []
 
+        aws_domain_name_servers = (aws.domain_name_servers || []).sort
         if @domain_name_servers.sort != aws.domain_name_servers.sort
```

Comment:
Yup. Was trying to catch a NilClass exception... and then left it in 2 more times